### DISCUSSION TITLE
Broken ExplorerPatcher install

### DISCUSSION THREAD

**Discussion by alexgm12** [valinet/ExplorerPatcher#3040] [Q&A]
I was having some issues with the File Explorer showing up every time I entered or exited fullscreen apps, so I tried uninstalling ExplorerPatcher to troubleshoot it. The details from here are hazy because I've been trying to work on this off and on for a few weeks. It gave me an error when I tried. From there I'm pretty sure I tried updating, manually deleting the folder in Program Files and installing, and one or two other things. Totally possible that I made something worse in the process.

Ultimately, I've tried installing the most recent version (22621.3296.64.1), however when I do, it doesn't appear to be completing the installation. The start menu works as normal, however when I try to go to the Properties right click menu, it gives me the error that ep_gui.dll is missing from the ExplorerPatcher Program Files folder. When I look in that folder, the only file there is WebView2Loader.dll. If I restart the computer, the start menu reverts back to the Windows 11 default. My antivirus software doesn't show that it's messing with any of the files.
My next hunch would be deleting registry keys associated with ExplorerPatcher and try to reinstall, but since I'm not too knowledgeable on how those work, I figured I'd come here before I start chewing on the wires any further.

I've also tried downloading ep_setup.exe, renaming it to ep_uninstall.exe and running it, but this just seems to crash explorer.exe.  

Any guidance on this?

**Comment by pyrates999** [comment]
try doing an uninstall of explorer patcher instead: https://github.com/valinet/ExplorerPatcher/wiki/Installer-How-To#how-to-uninstall

**Reply by alexgm12** [reply]
I have tried uninstalling (using the method of renaming ep_setup.exe to ep_uninstall.exe), but it just appears to crash explorer.exe (black screen, have to restart or open task manager and run explorer manually to do anything further). I can't uninstall from the menu since it doesn't show up in the first place (gives me the missing gui dll error I mentioned).

**Reply by pyrates999** [reply]
Try creating a new user, login as that user, and see if EP doesn't crash.  If EP doesn't crash, then it's something in your users directory that is causing it.

**Reply by alexgm12** [reply]
No luck with that, the files are still missing even on another user account and installing/uninstalling still lead to the same issues.

**Comment by pyrates999** [comment]
You may be talking about this: https://github.com/valinet/ExplorerPatcher/issues/2543

**Reply by alexgm12** [reply]
Yes, it sounds like a similar issue to the one I was originally troubleshooting, but I'm just trying to get it to install properly again before I tackle that problem.

**Comment by Amrsatrio** [comment]
I would try shift+deleting the C:\Program Files\ExplorerPatcher and %APPDATA%\ExplorerPatcher folder and these other files:

```
// C:\Windows
        // + dxgi.dll

        // C:\Windows\SystemApps\Microsoft.Windows.StartMenuExperienceHost_cw5n1h2txyewy
        // + dxgi.dll
        // + wincorlib.dll
        // + wincorlib_orig.dll (symlink)
        // - AppResolverLegacy.dll
        // - StartTileDataLegacy.dll
        // - Windows.UI.ShellCommon.pri
        // - en-US\StartTileDataLegacy.dll.mui
        // - pris2\Windows.UI.ShellCommon.en-US.pri

        // C:\Windows\SystemApps\ShellExperienceHost_cw5n1h2txyewy
        // + dxgi.dll
```

And then trying to install normally and then uninstalling.

**Reply by alexgm12** [reply]
This lead to some progress; I was able to actually bring up the Explorer Patcher menu and make changes but it stopped working soon after and the files disappeared again, I think after restarting.  I'm going to try again and do some more testing

**Reply by Naabo** [reply]
I just ran into the same issue when I did an update. Did you ever find a solution?

**Comment by rustamklekit** [comment]
Your antivirus software is likely deleting `ep_gui.dll` and other files necessary for Explorer Patcher to work.
